### DISCUSSION TITLE
Is possible to open a paper in full screen mode?

### DISCUSSION THREAD

**Discussion by LeleDallas** [clientIO/joint#2739] [Q&A] [answered]
### Introduction

I need to open a paper in full screen mode. How can I achieve that behavior? 

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Yes, you can use [toggleFullScreen()](https://docs.jointjs.com/api/util/#togglefullscreen) utility.

```ts
util.toggleFullScreen(paper.el);
```

**Reply by LeleDallas** [reply]
How can I trigger it on opening the website in a React project?

**Reply by kumilingus** [reply, marked as answer]
I don't think that's possible for security reasons: https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullscreen#security

**Reply by LeleDallas** [reply]
> I don't think that's possible for security reasons: https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullscreen#security

It says:
```
If permission to enter full screen mode is granted, the returned Promise will resolve and the element will receive a fullscreenchange event to let it know that it's now in full screen mode
```

Anyway, is it difficult to implement this feature? 
Like passing a boolean option `fullscreen` at the paper creation

**Reply by kumilingus** [reply]
It also says:

> [Transient user activation](https://developer.mozilla.org/en-US/docs/Web/Security/User_activation) is required. The user has to interact with the page or a UI element in order for this feature to work.

Thus, it cannot be switched to full screen without prior user interaction.

**Reply by LeleDallas** [reply]
> It also says:
> 
> > [Transient user activation](https://developer.mozilla.org/en-US/docs/Web/Security/User_activation) is required. The user has to interact with the page or a UI element in order for this feature to work.
> 
> Thus, it cannot be switched to full screen without prior user interaction.

Right
